packages/ssi-types/src/mapper/credential-mapper.ts

```diff
@@ -217,13 +227,19 @@ export class CredentialMapper {
       isJwtEncoded || isJwtDecoded
         ? CredentialMapper.jwtDecodedPresentationToUniformPresentation(decoded as JwtDecodedVerifiablePresentation, false)
         : (decoded as IVerifiablePresentation)
+
+    // At time of writing Velocity Networks does not conform to specification. Adding bare minimum @context section to stop parsers from crashing and whatnot
+    if (!uniformPresentation['@context']) {
+      uniformPresentation['@context'] = ['https://www.w3.org/2018/credentials/v1']
+    }
+
     uniformPresentation.verifiableCredential = uniformPresentation.verifiableCredential?.map((vc) =>
       CredentialMapper.toUniformCredential(vc, opts)
     ) as IVerifiableCredential[] // We cast it because we IPresentation needs a VC. The internal Credential doesn't have the required Proof anymore (that is intended)
     return uniformPresentation
   }
 
-  static jwtEncodedCredentialToUniformCredential(jwt: string, opts?: { maxTimeSkewInMS?: number }): IVerifiableCredential {
+  static jwtEncodedCredentialToUniformCredential(jwt: string, opts?: { maxTimeSkewInMS?: number }): ICredential {
```

Comment:
@nklomp not sure of I can do this here. The credential provided by Velocity does not have a proof section.

Reply:
No don't. The code adds the JWT as proof when it converts/decodes it to the Uniform VC (or it should do that)